tests/libnet/interface.go

```diff
@@ -37,6 +38,28 @@ func InterfaceExists(vmi *v1.VirtualMachineInstance, interfaceName string) error
 	return nil
 }
 
+func AddIPAddress(vmi *v1.VirtualMachineInstance, interfaceName, interfaceAddress string) error {
+	setStaticIpCmd := fmt.Sprintf("ip addr add %s dev %s\n", interfaceAddress, interfaceName)
+	err := exec.CommandOnVMI(vmi, setStaticIpCmd)
+
+	if err != nil {
+		return fmt.Errorf("could not configure address %s for interface %s on VMI %s: %w", interfaceAddress, interfaceName, vmi.Name, err)
+	}
+
+	return setInterfaceUp(vmi, interfaceName)
```

Comment:
Why setting the interface to up is part of this method?

Reply:
> As you decided to move this helper to a library, you encourage all to use it and therefore its correctness and relevance is more important than before.
> When it was in a specific test file, it was limited to that usage and therefore non generic additions, like the side-effect where reasonable. Now I do not think it is reasonable.

Any test file under tests/network can use it and it is not visible as importing a package, you can look at the diff and see it already used in tree places.
Ensuring the interface up is already done today (where `configInterface` is called) and has negligible cost.

From what I know (and verified) when spinning up a VM with secondary interface, on Fedora the interface is UP while on Alpine its state is DOWN.
I guess this is due to network-manager absence on Alpine.

The function is used to configure Alpine VMs, I can rename it to mention its for Alpine, would that be sufficient?

Reply:
My intention was that the setting the interface to up should be called separately where needed. I.would except the interface state to be changed to up after setting a static ip.
I don't expect you to debug as part of this pr all the callers and verify where it is needed and where not, you can just call the interface up after each of the current method calls.